src/repositories/northstar.js

```diff
@@ -183,6 +183,48 @@ export const updateEmailSubscriptionTopic = async (
   return null;
 };
 
+/**
+ * Update a user's list of causes they are interested in.
+ *
+ * @param {String} id
+ * @param {Cause} cause
+ * @param {Boolean} interested
+ * @param {Object} options
+ */
+export const updateCausePreferences = async (
+  id,
+  cause,
+  interested,
+  options,
+) => {
+  logger.debug('Updating cause for user in Northstar', {
+    id,
+    cause,
```

Comment:
Might be handy to include `interested` here too for debugging!

```suggestion
    cause,
    interested,
```